packages/backend/src/services/SpaceService/SpaceService.ts

```diff
@@ -15,6 +15,12 @@ type Dependencies = {
     spaceModel: SpaceModel;
 };
 
+const hasSpaceAccess = (space: Space, userUuid: string): boolean =>
+    // TODO enable this once UI for space access is done
+    // return !space.isPrivate || space.access.find(userAccess => userAccess.userUuid === userUuid && userAccess.role !== null ) !== undefined
+
+    true;
```

Comment:
We can uncomment this because spaces are still being created as public so it will return true for all spaces until we change the line in the spaces model:
```is_private: false, // TODO change to true once we support private spaces in the UI,```